upgradability/set_code_hash/transparent_voting/README.md

```diff
@@ -0,0 +1,35 @@
+## Transparent voting example
+
+This smart contract is part of an example showing how we can write upgradable smart contracts in ink! using `set_code_hash`.
+
+See `voting` example first.
+
+### Description
+
+This example provides a very similar functionality to the contract from `voting` example.
+It provides a simple voting interface which has been extended with `votes_for_0` and
+`votes_for_1`, which return current number of votes for the selected option.
+
+The point of this example is to show how to perform a smart contract upgrade using `set_code_hash` method
+in a case in which storage layout of the new contract does not differ from the old one.
+
+### Upgrading `voting` smart contract: upgrade without changing storage layout
+
+This contract provides additional functionality when compared to `voting` contract.
+We might want to add the functionality of reading vote numbers, but we might have already
+deployed `voting`, and gathered some votes, so we don't want to lose our voting data.
+
+To upgrade to the new logic, it is enough to call method `set_code` with the hash of the new code.

```

Comment:
please specify to `voting::set_code` - `set_code` alone may lead to a little confusion, especially in substrate environment :)